Makefile

```diff
@@ -72,6 +72,11 @@ flask: | lib/sqlalchemy
 lib/redis:
 	$(BIN)/pip install redis
 
+db-setup:
+	psql -U postgres -c "create database django_app;"
+	psql -U postgres -c "create user django_app with password 'django_app';"
+	psql -U postgres -c "grant all privileges on database django_app to django_app;"
```

Comment:
if you create the user first, then you could set the owner of the django_app db to the django_app user and skip the grant?

e.g. CREATE DATABASE django_app OWNER django_app

Reply:
Huh, TIL, nice